enospc_test.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2017 ScyllaDB
+
+from avocado import main
+from sdcm.tester import ClusterTester
+from sdcm.nemesis import EnospcMonkey
+
+
+class EnospcTest(ClusterTester):
+    """
+    Repeatedly cost rest space to trigger ENOSPC error for scylla,
+    and release space for recover scylla.
+    """
+    def test_enospc_nodes(self):
+        self.db_cluster.add_nemesis(nemesis=EnospcMonkey,
+                                    loaders=self.loaders,
+                                    monitoring_set=self.monitors)
+
+        self.db_cluster.wait_total_space_used_per_node()
+        # run a write workload
+        base_cmd_w = ("cassandra-stress write no-warmup cl=QUORUM duration=60m "
+                      "-schema 'replication(factor=3)' -port jmx=6868 "
+                      "-mode cql3 native -rate threads=100 -errors ignore "
+                      "-pop seq=1..10000000")
+        stress_queue = self.run_stress_thread(stress_cmd=base_cmd_w, stress_num=2, keyspace_num=1)
+
+        for i in range(10):
+            # Sleep 3 minutes before enospc nemesis
```

Comment:
ok, but I would use the nemesis interval doing it... each time on different target node..
Also, I don't think we need 10 times.